packages/varlet-ui/src/collapse-item/CollapseItem.vue

```diff
@@ -59,12 +59,14 @@ export default defineComponent({
     const showContent = ref(false)
     const contentEl = ref<HTMLDivElement | null>(null)
     const name = computed(() => props.name)
+    const disabled = computed(() => props.disabled)
     const { index, collapse, bindCollapse } = useCollapse()
     const { offset, divider, elevation, updateItem } = collapse
 
     const collapseItemProvider: CollapseItemProvider = {
       index,
       name,
+      disabled,
```

Comment:
why computed? According to my experience props are reactive.

Reply:
![image](https://github.com/varletjs/varlet/assets/81673017/d15b942e-4f92-44b5-b59e-67ad1926be71)